alica_tests/src/test/etc/hairy/Alica.yaml

```diff
@@ -88,3 +88,6 @@ Alica:
     SeedTTL4Usage: 5000
     MaxSolveTime: 10
     MaxFunctionEvaluations: 100000000
+
+  CustomerLibrary:
```

Comment:
@triccyx suggestion from @Dale-Koenig 
use LD_LIBRARY_PATH if you are loading using boost 
use ROS path if you are loading using ros 
since this PR is using boost, you can start with using LD_LIBRARY_PATH